src/mixins/document-mixin.ts

```diff
@@ -17,14 +18,13 @@ export default class DocumentMixin extends Vue {
     }
   }
 
-  private pdfjsLib: any
+  pdfjsLib: any
 
-  created () {
+  async created () {
     // NB: we load the lib and worker this way to avoid a memory leak (esp in unit tests)
-    // NB: must use require instead of import or this doesn't work
     // NB: must use legacy build for unit tests not running in Node 18+
-    this.pdfjsLib = require('pdfjs-dist/legacy/build/pdf.js')
-    this.pdfjsLib.GlobalWorkerOptions.workerSrc = require('pdfjs-dist/legacy/build/pdf.worker.entry')
+    this.pdfjsLib = pdfjs
+    this.pdfjsLib.GlobalWorkerOptions.workerSrc = await import('pdfjs-dist/legacy/build/pdf.worker.entry')
```

Comment:
Please test this and/or bring it to Riyaz's attention for QA.

Reply:
Tested, works well! I'll also add a note for Riyaz 👍 

If you want Sev, here's a link if you quickly want to give it a try: https://business-create-dev--pr-556-nilgmamu.web.app/incorporation-memorandum?id=TA7zV53Hu5

Reply:
Thanks. I just tried it. It seems to work fine for a proper PDF file, ie:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/815b8292-c04d-42a2-8543-ad1419bbb333)

However, it report success when selecting an invalid file (though there are console errors):

![image](https://github.com/bcgov/business-create-ui/assets/10002889/df3c9da4-82c1-426e-9a14-b9b50d5df7eb)

Reply:
What do you mean Sev? Did you select a non-pdf file? I think if you select a non-pdf file, the select button shows "File must be a pdf". It'll also show that the step is incomplete in Review and Confirm. Is that fine?

Reply:
Yes, I selected a non-PDF file.

Ah, OK, so even the current Create UI doesn't show an error message if the file is invalid. I was thinking about the PDF upload in Filings UI (staff filings), which I had refactored and which does show an error message, ie:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/df9af247-572c-4f39-9e24-041d209b7067)

But changing this over is out of scope for this ticket, so I guess this is OK for now -- just as long as users cannot upload and file with an invalid PDF file.